migrations/805-add-addon-is_listed.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE addons
```

Comment:
I think in the past a ALTER on the addons table will take about 10-15 minutes. We can test on dev and stage to have a better idea how long it takes.